### PR TITLE
improve short format for balance credits >1K and >1M

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: src/util/format-credits.js [hunk 1]
```diff
@@ -5,6 +5,21 @@ export function formatCredits(amount, precision) {
     .replace(/\.?0+$/, '');
 }
 
+export function formatBigNumberCredits(amount, precision) {
```
Comment (kauffj) [inline src/util/format-credits.js]: Would it make more sense for this functionality to be a parameter to formatCredits rather than a standalone function?

Comment (akinwale) [reply src/util/format-credits.js]: I was trying to avoid having to update all the references to the method, as there are quite a few.

Comment (kauffj) [reply src/util/format-credits.js]: Why would you have to update the references if you add an optional 3rd parameter?

Comment (akinwale) [reply src/util/format-credits.js]: I suppose I wouldn't have to, now that I think about it. I should be able to roll the functionality into the `formatCredits` method.

**Pull request merged**